README.md

```diff
@@ -233,6 +235,28 @@ Need to setup Flatpak (https://flatpak.org/setup/):
 * Chrome OS
 
 
+Systemd
+-------
+If your distribuition is running under systemd and you want to generate and upload
+hw-probe report automatically, please install as root:
+
+    cp -a periodic/* $(systemdsystemunitdir)/
```

Comment:
Please specify which files to copy or create a subdirectory for systemd-based systems.

Reply:
Done